index.html

```diff
@@ -169,18 +203,19 @@ <h4 class="languages_heading">Languages</h4>
                     <span>CSS</span>
                 </li>
 
-            </ul>
-        </div>
+        </ul>
+        
         <div class="languages_frameworks">
             <h4>Frameworks</h4>
-            <img src="./images/Union.png" alt="icon">
+            <img src="./images/Union.png" alt="icon" class="aero">
         </div>
       <hr>
```

Comment:
- [ ] Your design is looking great! However, it is important to fix this line under frameworks that is breaking the design:

Template: 
![Captura de pantalla 2022-03-26 a la(s) 11 03 44 a  m](https://user-images.githubusercontent.com/83717285/160248278-8695ce1f-c772-4fce-afd1-0198fecc4bac.png)

Your design:
![Captura de pantalla 2022-03-26 a la(s) 11 04 05 a  m](https://user-images.githubusercontent.com/83717285/160248290-7572104a-0089-4e49-956e-30c701c19f19.png)